lib/gantree/release_notes.rb

```diff
@@ -78,11 +78,15 @@ def execute(cmd)
     end
 
     def notes
-      compare = "#{previous_sha}...#{current_sha}"
-      notes = <<-EOL
+      if previous_sha
+        compare = "#{previous_sha}...#{current_sha}"
```

Comment:
I don't think this will work, because it still calls previous_sha, which will try to do a split on previous_tag, but that might be null. So I think you also need to modify previous_sha to check for null before attempting to split.